hack/manifest-gen/README.md

```diff
@@ -0,0 +1,44 @@
+ECK Manifest Generator
+======================
+
+Utility for generating ECK installation manifests.
+
+Quickstart
+----------
+
+NOTE: Requires Docker and Make to be installed on your computer.
+
+Cluster-wide installation with access to all namespaces:
+
+```shell
+make docker-gen-global | kubectl apply -f -
```

Comment:
The script is simply there to update the chart with the latest CRDs and version. People can use the CLI directly if they wish but I think the main audience for the tool is non-developers who would prefer a simpler command to run rather than a complicated compilation + run recipe.

Reply:
> the main audience for the tool is non-developers

But in that case they would probably run either form a released docker image or released binary directly, not from cloning the github repo and running a make command?